src/utils/compose.js

```diff
@@ -6,5 +6,9 @@
  * left. For example, compose(f, g, h) is identical to arg => f(g(h(arg))).
  */
 export default function compose(...funcs) {
-  return arg => funcs.reduceRight((composed, f) => f(composed), arg)
+  return (...args) => {
+    return funcs.length ?
```

Comment:
It is a little hard to grasp what this is doing at first.
Maybe re-writing this as an `if` statement would make this function a bit easier to follow.